### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Andremoniy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+# Cypress Visual Regression with Resemble.JS
+
+[![npm](https://img.shields.io/npm/v/cypress-visual-regression-resemble-js)](https://www.npmjs.com/package/cypress-visual-regression-resemble-js)
+
+[![github actions](https://github.com/Andremoniy/cypress-visual-regression-resemble-js/workflows/Continuous%20Integration/badge.svg)](https://github.com/Andremoniy/cypress-visual-regression-resemble-js/actions)
```
Comment (PpBoyington) [inline README.md]: This image does not display itself ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  cy.get('H1').contains('Login');
+  cy.compareSnapshot('login', 0.0);
+  cy.compareSnapshot('login', 0.1);
+});
+```
+
+You can target a single HTML element as well:
+
+```js
+cy.get('#my-header').compareSnapshot('just-header')
```
Comment (PpBoyington) [inline README.md:183]: Ah well I didn't know we could do that, there is a section of the plugin that I developed to be able to do so with an additional param, I will have to delete that.

File: docker/sample_application/test.sh [hunk 3]
```diff
@@ -29,5 +29,6 @@ echo "Running cypress installation and test"
 
 cd sample_application_cypress
 npm i
+npm run cypress:run || true
```
Comment (PpBoyington) [inline docker/sample_application/test.sh:32]: what's this ?

**Review by PpBoyington — COMMENTED** [review]
small comments

File: README.md (hunk 1 above)
Comment (Andremoniy) [reply README.md]: Let's say: it is not ready yet. I will remove for now and investigate more.

**Review by PpBoyington — APPROVED** [review]
GJ!

**Pull request merged**